integration_tests/src/main/python/map_test.py

```diff
@@ -475,6 +502,14 @@ def test_map_element_at_ansi_fail(data_gen):
             conf=test_conf,
             error_message=message)
 
+@pytest.mark.skipif(is_before_spark_340(),
```

Comment:
same `and not is_databricks113_or_later()` question here.

Reply:
These 2 tests fail on db11.3, 
```
FAILED ../../src/main/python/map_test.py::test_element_at_map_string_col_keys_ansi_null[Map(String(not_null),String)] - py4j.protocol.Py4JJavaError: An error occurred while calling o70730.collect...
FAILED ../../src/main/python/map_test.py::test_map_element_at_ansi_null[Map(String(not_null),String)] - py4j.protocol.Py4JJavaError: An error occurred while calling o76588.collect...
```

Reply:
I need to create a separate test case for bd11.3 I guess. Because it seems like that db11.3 removes `spark.sql.ansi.strictIndexOperator` but still throws exceptions. (I need to double confirm this)

Reply:
Hi @sameerz @jlowe @GaryShen2008, As Gary mentioned, should we create another PR to fix the behavior of `invalid access on map` on DB11.3 and merge it in the branch 23.02?

Reply:
We discussed this and decided it can go in 23.04, since it is a fix for a specific mode of Spark SQL (ANSI) and the error does not result in data corruption.